presentation/reward/src/main/res/values/strings.xml

```diff
@@ -35,4 +35,5 @@
     <string name="menu_todoist">Todoist</string>
     <string name="section_other">Other</string>
     <string name="menu_logout">Logout</string>
+    <string name="unnecessary_string">Hoge</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/4b56ce12e1b77906552fadb7898acd26503a2fe3/presentation/reward/src/main/res/values/strings.xml#L38"></span>The resource <code>R.string.unnecessary_string</code> appears to be unused</td>
    </tr>
  </tbody>
</table>